### ISSUE TITLE
Doubts about the usage of the code

### ISSUE THREAD

**Issue by MRCSDSLD** [cberner/deepstep#23]
Hi, your code is amazing, is very simple and funcional. I am very beginner in python and Deep Learning, so I have some doubts like, whats the hyperflow do? 
What is a good seed file? A skrilex song? 
What type of midi file is good for training the model? I used this [https://www.reddit.com/r/WeAreTheMusicMakers/comments/3ajwe4/the_largest_midi_collection_on_the_internet/] dataset, but the index_training_data.py just picked some files, about 100 or less in midi files avaiables. 
When  I used this:([https://www.reddit.com/r/WeAreTheMusicMakers/comments/3anwu8/the_drum_percussion_midi_archive_800k/])  the index_training_data pick more than 21000 files to training. Is it a good amount to training? 
Is it possible to recofigure the index_training_data to pick more files in the avaibles?
What the optimizer_hyperparameters perform? I should be used this on the selectioned training dataset or in the data to be selectioned? 
Thank you for your attention
